how-to-capture-fiddler-trace.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to capture Fiddler trace
+description: Step by step tutorial for capturing HTTP/HTTPS traffic with Fiddler
+type: how-to
+page_title: How to capture Fiddler trace
+slug: capture-fiddler-trace
+position: 99
+tags: fiddler, how-to
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+
+> **Service:** 3rd party tool
+> 
+> **Other stuff:** Other stuff here
+
+## Description
+This KB provides general steps for capturing HTTP/HTTPS traffic with Fiddler and saving it as a log file on the local machine. Fiddler traces are often requested by Telerik Platform support engineers, as they help tremendously in identifying issues with the Telerik Platform services on the client side. 
```

Comment:
Active voice is usually the recommendation. For example, "The Telerik Platform support engineers often request Fiddler traces when they need to investigate \<such and such\> reports."

I would probably go with something like this for this paragraph:

"This article provides the general steps for capturing HTTP/HTTPS traffic with Fiddler and saving it as a log file on your local machine. When you submit reports for X,Y,Z-related issues, the Telerik Platform support engineers are very likely to request a detailed Fiddler log to aid them in the investigation of the issue."